client/src/components/RockPaperScissor.vue

```diff
@@ -0,0 +1,208 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useGameStore, Selections } from '../stores/game';
+import { usePopUpStore } from '../stores/popup';
+
+const gameStore = useGameStore();
+const popUpStore = usePopUpStore();
+
+const userHasSelected = computed(() => {
+  return gameStore.userSelection != 3 ? true : false;
+});
+const botIsMakingSelection = computed(() => {
+  return gameStore.userSelection != 3
+    ? gameStore.botSelection === 3
```

Comment:
```suggestion
const userHasSelected = computed(() => {
  return gameStore.userSelection != Selections.none ? true : false;
});
const botIsMakingSelection = computed(() => {
  return gameStore.userSelection != Selections.none
    ? gameStore.botSelection === Selections.none
```

Proposing this change as it would improve readability
(Had to check the enum in order to understand the comparison need)